index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <a href="#" id="logo">My Logo</a>
+            <nav>
+                <i class="fa fa-bars fa-lg"></i>
+            </nav>
+        </header>
+        <section class="main-content">
+            <div class="main-content-container">
+                <div>
+                    <h1>I'm Mariane</h1>
+                    <h1>Glad to see you!</h1>
```

Comment:
- [ ] `h1` tag should not be used twice in a webpage so that it is not semantically correct. You can consider using `<br/>` tag to break a line. Such as:

```suggestion
<h1>I'm Mariane <br />
          Glad to see you!!</h1>
```
- Kindly fix this issue.